tests/jobs/design2params/test_design2params.py

```diff
@@ -1,20 +1,20 @@
 import filecmp
 import logging
 import os
-from distutils.dir_util import copy_tree
+import shutil
```

Comment:
Why this changes are part of this PR? I think they both work with Pythin3.6+, do they not?

Reply:
`distutils` is deprecated in Python 3.10